### PR TITLE
Topic resteasy queryparams defaultvalue

### PR DESCRIPTION
**Pull request by andreaformica**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming langauge.

### Description of the PR
The mustache template that I proposed in the previous PR seemed to contain some errors. I changed the queryParams template again (always for JavaJaxRs resteasy ), hopefully this one is fine.

### CODE REVIEW

File: modules/swagger-codegen/src/main/resources/JavaJaxRS/resteasy/queryParams.mustache [hunk 1]
```diff
@@ -1 +1 @@
-{{#isQueryParam}}{{#useBeanValidation}}{{>beanValidationQueryParams}}{{/useBeanValidation}} @QueryParam("{{baseName}}") {{{dataType}}} {{paramName}}{{/isQueryParam}}
\ No newline at end of file
+{{#isQueryParam}}{{#useBeanValidation}}{{>beanValidationQueryParams}}{{/useBeanValidation}}{{#defaultValue}} @DefaultValue("{{{defaultValue}}}"){{/defaultValue}} @QueryParam("{{baseName}}") {{{dataType}}}{{paramName}}{{/isQueryParam}}
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/JavaJaxRS/resteasy/queryParams.mustache]:
```
{{{dataType}}}{{paramName}}
```
should be
```
{{{dataType}}} {{paramName}}
```
with a space between.

Comment (andreaformica) [reply modules/swagger-codegen/src/main/resources/JavaJaxRS/resteasy/queryParams.mustache]: I just committed the change you mention @wing328 .

**Comment by wing328** [comment]
@andreaformica I did some tests and `mvn test` ran without issue for the updated RESTEasy samples (except that I need to remove the newline at the end of the file)

**Comment by wing328** [comment]
FYI. Newline removed via b259f3b

**Pull request merged**